nrt/data/DynaFlow/GeneratorPVTfo_rpcl_rpcl2/GeneratorPVTfo_rpcl_rpcl2.jobs

```diff
@@ -0,0 +1,31 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<!--
+    Copyright (c) 2022, RTE (http://www.rte-france.com)
+    See AUTHORS.txt
+    All rights reserved.
+    This Source Code Form is subject to the terms of the Mozilla Public
+    License, v. 2.0. If a copy of the MPL was not distributed with this
+    file, you can obtain one at http://mozilla.org/MPL/2.0/.
+    SPDX-License-Identifier: MPL-2.0
+
+    This file is part of Dynawo, an hybrid C++/Modelica open source suite of
+    simulation tools for power systems.
+-->
+<dyn:jobs xmlns:dyn="http://www.rte-france.com/dynawo">
+  <dyn:job name="GeneratorPVTfo_rpcl_rpcl2">
+    <dyn:solver lib="dynawo_SolverSIM" parFile="GeneratorPVTfo_rpcl_rpcl2.par" parId="Solver"/>
```

Comment:
The files could be renamed. I propose GeneratorPVTfoRpclTimeline.par (same for the others)